index.js

```diff
@@ -2,36 +2,36 @@
 /* global process */
 'use strict';
 
-let path = require('path');
-let Funnel = require('broccoli-funnel');
-let MergeTrees = require('broccoli-merge-trees');
+const path = require('path');
+const Funnel = require('broccoli-funnel');
+const MergeTrees = require('broccoli-merge-trees');
+const map = require('broccoli-stew').map;
 
 module.exports = {
 
   name: 'ember-gestures',
 
-  included: function (app) {
+  included(app) {
     this._super.included.apply(this, arguments);
 
     // see: https://github.com/ember-cli/ember-cli/issues/3718
     if (typeof app.import !== 'function' && app.app) {
       app = app.app;
     }
 
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      app.import('vendor/hammer.js');
-    }
+    app.import('vendor/hammer.js');
```

Comment:
Yes, you do. You can only `import` from `bower_component` and `vendor`, that's why the file from `node_modules` needs to be moved to the vendor tree. But you still have to import it.